src/__tests__/input.test.js

```diff
@@ -1,9 +1,17 @@
 const Input = require("../input")
 
-test("valid input", () => {
+test("valid input - no version", () => {
   const input = new Input("owner", "repo", ["package"], ".*", "2", "token")
 
   expect(input.owner).toBe("owner")
+  expect(input.version).toBeNull()
+})
+
+test("valid input - with version", () => {
+  const input = new Input("owner", "repo", ["package"], ".*", "2", "token", "foo-bar")
```

Comment:
I feel like `versionPattern` and `keep` should be set to `null` when `version` is set and the other way around. This way we can maybe also move the validation to the `Input` class (and the default parameters handling also?). Seems more clean because that enforces the class to always have valid state.